src/transformers/models/auto/modeling_auto.py

```diff
@@ -509,6 +512,7 @@
         ("pegasus_x", "PegasusXForConditionalGeneration"),
         ("plbart", "PLBartForConditionalGeneration"),
         ("prophetnet", "ProphetNetForConditionalGeneration"),
+        ("switch_transformers", "SwitchTransformersForConditionalGeneration"),
```

Comment:
Please also add the default to `T5Tokenizer` in the tokenization_auto module.

Reply:
Fixed in ccc28a9f1652aaedf9590842b1d624cc697acd58